kcidb/test_db.py

```diff
@@ -1059,15 +942,15 @@ def test_query(empty_database):
     }
     assert client.query(ids=dict(incidents=["_:3"]),
                         parents=True, children=True) in [
-        {
-            **unambigious_result,
-            "incidents": [incident_x, incident_y],
-        },
-        {
-            **unambigious_result,
-            "incidents": [incident_y, incident_x],
-        },
-    ]
+               {
+                   **unambigious_result,
+                   "incidents": [incident_x, incident_y],
+               },
+               {
+                   **unambigious_result,
+                   "incidents": [incident_y, incident_x],
+               },
+           ]
```

Comment:
Please don't post unrelated whitespace changes. Here, and above. If you need to change this for something, e.g. to make CI work, make a separate commit doing it, and provide explanation in the commit message. Thanks!